plugins/lookup/credstash.py

```diff
@@ -79,6 +79,7 @@
   _raw:
     description:
       - value(s) stored in Credstash
```

Comment:
```suggestion
      - Value(s) stored in Credstash.
```
the same, feel free to ignore my comment if all stuff is like this there